package.json

```diff
@@ -52,7 +52,8 @@
     "reconnecting-websocket": "^4.4.0",
     "redux": "^4.0.5",
     "redux-thunk": "^2.3.0",
-    "regenerator-runtime": "^0.13.5"
+    "regenerator-runtime": "^0.13.5",
+    "web-streams-polyfill": "^2.1.1"
```

Comment:
this should be a dev dependency
`npm remove web-streams-polyfill && npm install -D web-streams-polyfill`